pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
```

Comment:
`wheel` is from some outdated documentation; you probably don't need it for your build. It should be automatically installed where necessary.

Reply:
@william-silversmith Here's a second question that you may be able to answer.  I unfortunately have minimal knowledge of Python, though I suspect it is related to our zfpy-wheels repo for building and uploading wheels to PyPI.